media/base/media_client.h

```diff
@@ -56,6 +62,13 @@ class MEDIA_EXPORT MediaClient {
   // Optionally returns audio renderer algorithm parameters.
   virtual absl::optional<::media::AudioRendererAlgorithmParameters>
   GetAudioRendererAlgorithmParameters(AudioParameters audio_parameters) = 0;
+
+#if BUILDFLAG(IS_COBALT)
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+ private:
+  DecoderBufferAllocator decoder_buffer_allocator_;
```

Comment:
Where and how will we use `decoder_buffer_allocator_`?

Reply:
TODO: we should add Suspend() and Resume() as we did on Cobalt C25 in cobalt/media/media_module.cc.

Reply:
From b/326497953, supporting Suspend() and Resume() is not MVP. We should add a TODO here.